src/app/session.service.ts

```diff
@@ -1,15 +1,35 @@
 import { Injectable } from '@angular/core';
 import { Observable } from 'rxjs';
 import { Http } from '@angular/http';
+import { GlobalSettingsService } from './globalSettings.service';
 
 @Injectable()
 export class SessionService {
+  config;
+  constructor(private http: Http, private globalSettingsService: GlobalSettingsService ) {
+    this.config = globalSettingsService.get();
+  }
+
+  create(): Observable<any> {
+    return this.http.post(`${this.config.apiUrl}/sessions`, {});
+  }
+
+  get(sessionId: string): Observable<any> {
+    return this.http.get(`${this.config.apiUrl}?sessionId=${sessionId}`);
+  }
+
+  end(sessionId: string): Observable<any> {
```

Comment:
The delete call should still return a http status code. We can use this to then handle errors

Ill link a RFC standard for what a method should return in slack